lib/vocab/csw_terms.rb

```diff
@@ -0,0 +1,7 @@
+require 'rdf'
+module Vocab
+  class CswTerms < RDF::Vocabulary('http://www.opengis.net/def/serviceType/ogc/csw/2.0.2#')
+    # No terms have been used within any Concerns
```

Comment:
I assumed that it would be included as the support for geospatial metadata were extended for future cases (unfortunately, it has not been included within RDF::Vocab, and I have created #65 in order to request that this possibility be explored).  I defer to @drh-stanford, @johnhuck, and @yulgit1 on this topic.

If this is not the case, then I can remove this for the time being.